about.html

```diff
@@ -0,0 +1,148 @@
+            <li><a href="#main" onclick="closeNav()">Program</a></li>
+            <li><a href="#main1" onclick="closeNav()">Logo</a></li>
+            <li><a href="#past" onclick="closeNav()">Past Show</a></li>
+            <li><a href="#contu" onclick="closeNav()">Contacts</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section id="main" class="main flex-center">
+        <h1>
+          Oscar 2023 awards <br />
+          winers & nominess
+        </h1>
+        <div class="mainp flex-center">
+          <p>
+            The Academy Awards, better known as the Oscars,[1] are awards for
+            artistic and technical merit for the global film industry. The
+            awards are regarded by many as the most prestigious, significant
+            awards in the entertainment industry in the United States.
+          </p>
+        </div>
+        <div class="maincont flex-center">
+          <p>
+            Please contact us per Email for nay further questions about
+            OSCARS-AWARDS-2023!
+          </p>
+          <span
+            ><ul>
+              oscar-support@ao.uscourts.gov
+            </ul></span
+          >
+        </div>
+      </section>
+
+      <section id="main1" class="main1 flex-center">
+        <div class="mainlogo flex-center">
+          <h2>
+            Oscar 2023 awards <br />
+            winers & nominess Logo
+          </h2>
+          <hr class="line" />
+          <p>
+            MGM art director Cedric Gibbons designed a statuette of a knight
+            standing on a reel of film gripping a crusader's sword.
+          </p>
+          <img src="assests/logo/logo.png" alt="logo-img" />
+        </div>
+      </section>
+
+      <section class="past flex-center" id="past">
+        <div class="pastaward flex-center">
+          <h2>See the past Oscar Awards</h2>
+          <hr class="line" />
+          <p>
+            Take a look at the past year Oscar Awards which took place in Los
+            Angeles California, U.S.
+          </p>
+          <div class="imgact flex-center">
+            <div class="past1">
+              <div class="pastimg flex-center">
+                <h2>2022</h2>
+                <p>Oscar Awards</p>
+              </div>
+            </div>
+            <div class="past2">
+              <div class="pastimg flex-center">
+                <h2>2022</h2>
+                <p>Oscar Awards</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="contact flex-center">
+      <h2>Contact</h2>
+      <hr class="line">
+      <div class="foot-cont">
+        <img src="assests/logo/logo.png" alt="img">
+        <p>THE 95TH ACADEMY AWARDS | 2023
+          Dolby Theatre at Ovation Hollywood
+          Sunday, March 12, 2023
+          Honoring movies released in 2022</p>
+      </div>
+      <p class="right">&copy; 2023 All Rights Reserved</p>
+    </footer>
```

Comment:
- [OPTIONAL] It is required to stick to the figma design as close as possible. The footer background color on about page is dark. Your design carries white background as opposed to the figma design. Kindly change this and ensure the height is also in line with the figma design. I marked this optional because it was not mentioned by the previous reviewer. I order to maintain consistency, I need to mark it optional but it is highly recommended that you attend to it. 

### Your Design
![your-about-footer-bg](https://user-images.githubusercontent.com/26597432/225531935-6078dc9f-f05b-469f-92d7-76d1d2a8ccfe.png)

### Figma Design
![temp-about-footer-bg](https://user-images.githubusercontent.com/26597432/225531972-4efa5cb6-4c11-4cf6-b7e7-4f2ce39064d6.png)

- [OPTIONAL] Your design misses one section of the content on the mobile version on about page. There's supposed to be partner section on mobile in about page. Kindly add that section. I marked this optional because it was not mentioned by the previous reviewer. I order to maintain consistency, I need to mark it optional but it is highly recommended that you attend to it. 

### Your Design
![your-about-mobile-footer](https://user-images.githubusercontent.com/26597432/225533194-14ae3c03-e6f5-4614-ae4a-93989c205364.png)

### Figma Design
![temp-mobile-footer](https://user-images.githubusercontent.com/26597432/225533266-75e768f3-c9ef-44e5-ac3f-5ead2ede8b78.png)